static/app/views/replays/detail/layout/splitDivider.tsx

```diff
@@ -0,0 +1,49 @@
+import {DOMAttributes} from 'react';
+import styled from '@emotion/styled';
+
+import {IconGrabbable} from 'sentry/icons';
+import {space} from 'sentry/styles/space';
+
+type Props = {
+  isHeld: boolean;
+  slideDirection: 'leftright' | 'updown';
+};
+
+const SplitDivider = styled(
+  ({isHeld: _a, slideDirection: _b, ...props}: Props & DOMAttributes<HTMLDivElement>) => (
+    <div {...props}>
+      <IconGrabbable size="sm" />
+    </div>
+  )
+)<Props>`
+  display: grid;
+  place-items: center;
+  height: 100%;
+  width: 100%;
+
+  user-select: ${p => (p.isHeld ? 'none' : 'inherit')};
+  background: ${p => (p.isHeld ? p.theme.hover : 'inherit')};
```

Comment:
This line is a little fix that I hadn't noticed for a long while, happy it's here now.